scripts/build/build.sh

```diff
@@ -22,6 +22,12 @@ JARS_DIR="$SCRIPTS_DIR/jars"
 DOCKER_DIR="$SCRIPTS_DIR/docker"
 VERSION_FILE="$SCRIPTS_DIR/templates/version.txt"
 
+if [ "$#" -lt 1 ]; then
+	MAVEN_PARAMS=("clean install -q -Pquick")
+else
+  MAVEN_PARAMS=("$@")
```

Comment:
Bit of a formatting issue with the indentation here

Reply:
Sorted, thanks!